liquibase-core/src/main/javacc/liquibase/util/grammar/SimpleSqlGrammar.jj

```diff
@@ -370,6 +370,8 @@ Built list from http://stackoverflow.com/a/37668315/45756
     "\u213C"-"\u213F",
     "\u2145"-"\u2149",
     "\u214E",
+    "\u2160"-"\u216F", //Roman numerals
```

Comment:
Should we just add the whole "Number Forms" block https://www.compart.com/en/unicode/block/U+2150 instead of these specifically?

Reply:
I though about it when I performed the change @nvoxland. In the end I didn't include the whole block because, on one hand, `SimpleSqlGrammar.jj` already included the values `\u2183` and `\u2184`; on the other,  I think the range from `\u2180` to `\u218B` defines symbols that are used less frequently.

In any case, I will be glad to include the whole "Number Forms" block if you prefer. Should I change it?